db/schema.rb

```diff
@@ -653,8 +654,8 @@
   create_table "provider_users_providers", force: :cascade do |t|
     t.bigint "provider_id", null: false
     t.bigint "provider_user_id", null: false
-    t.datetime "created_at", precision: nil, default: -> { "now()" }, null: false
-    t.datetime "updated_at", precision: nil, default: -> { "now()" }, null: false
+    t.datetime "created_at", precision: nil, default: -> { "CURRENT_TIMESTAMP" }, null: false
+    t.datetime "updated_at", precision: nil, default: -> { "CURRENT_TIMESTAMP" }, null: false
```

Comment:
Past Tomas vs future Tomas:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/7852/commits/9ddf1ca417d8904008a9770731d454bc5f13f473